Snakefile

```diff
@@ -1402,6 +1426,92 @@ rule generate_methods_differential_rnaseq:
         cp {output.methods} {wildcards.accession}-analysis-methods.tsv
         """
 
+rule deconvolution:
+    """
+    Runs three deconvolution tools (DWLS, FARDEEP, EpiDIS) for experiments selected in deconvolution.yaml.
+    """
+    conda: "envs/deconvolution.yaml"
+    log: "logs/{accession}-deconvolution.log"
+    #resources: mem_mb=get_mem_mb
+    resources: mem_mb=64000
+    threads: 16
+    input: 
+        fpkms="{accession}-fpkms.tsv.undecorated",
+	methods=get_methods_file,
+	#methods=rules.generate_methods_baseline_rnaseq.output,
+        sdrf=get_sdrf()
+    params:
+        signature_dir=config["deconv_ref"] + get_organism()
+    output:
+        proportions="{accession}-deconvolution.proportions.tsv",
+	methods="{accession}-analysis-methods.tsv",
+        results=(directory('Output/{accession}')),
+        splits=temp(directory('Tissue_splits/{accession}')),
+        scratch=temp(directory('scratch/{accession}'))
+    shell:
+        """
+        exec &> "logs/{wildcards.accession}-deconvolution.log"
+	set -b  #Notify of job termination immediately
+
+	echo "starting..."
+	# Split fpkms into organism parts
+	mkdir -p Tissue_splits/{wildcards.accession}
+	Rscript {workflow.basedir}/atlas-analysis/deconvolution/splitAndScale.R {input.fpkms} {input.sdrf} {wildcards.accession}
+
+	# list all files that FPKMs were split into
```

Comment:
what if there is no files here?
Should we implement a check here that exits Snakemake if there are no files?